rls-quick-start-guide.md

```diff
@@ -0,0 +1,58 @@
+- **`#help-and-advice`:** If there's not a more specific channel that's relevant to your question, you can always ask in this channel. 
+- **`#anonymous-help-and-advice`:** For various reasons, there are times where someone may not feel comfortable asking for advice publicly in RLS. This channel was created to support those times and allow members to create an anonymous top-level post using a channel workflow. The donwside is that it can remove the ability to have deeper discussions, since the original poster cannot respond anonymously in threads to participate.
+- **`#lobby-*`:** Lobby channels are public channels that enable members to request admission to private channels, which would otherwise be undiscoverable. Right now only two exist, `#lobby-adoption` and `#lobby-non-binary-and-women`, and you can see past discussions about the implementation and use of `#lobby-*` channels in `#rands-slack-culture`.
+- **`#daily-ftw`:** This is where we celebrate our wins, and it's a great channel to get started in! Post your own wins so we can cheer you on, and join the conversations on other people's posts!
+
+## Discovering New Channels
+
+Here are some channels you can join that are good for helping you discover other channels (plus some of these are just nice channels to be in!):
+
+- `#rands-slack-gratitude` 
+- `#dancing-penguin`
+- `#royquotes`
+- `#brightideas`
+- `#new-channels`
+
+## Reacji Channelers
+
+Several channels are set up with a workflow to automatically cross-post in a different channel based on specific emoji reactions (also known as reacjis). In Slack, you can use a slash command to view all of the channels with this cross-post workflow: `/reacji-channeler list`.
+
+## (Some) Reacjis and Their Meaning
+
+Some reacjis have a specific connotation that's not immediately clear. This is a non-exhaustive list of some that tend to be used more often:
+
+- **`:raccoon:`** has various forms, and is used to indicate that someone's contribution to a discussion is off-topic. This usage predates RLS, and more context can be found [https://techcrunch.com/2017/12/04/how-slack-uses-a-raccoon-to-keep-distractions-on-slack-at-bay/?guccounter=1](here).
+- **`:royr:` and `:ohroy:`** have two use cases. The first is to use the reacji channeler on something that Roy Rapoport (the most prolific poster on RLS) has said so that it automatically cross-posts into `#royquotes`. The second is to appreciate a bit of sarcasm or a dad joke in the spirit of Roy, even if he was not the one posting it. (The cross-posting action is still triggered in this use, it just wasn't the main intent.)
+- **`:shields-up`, `:shields-at50:`, `:shields-down:`, and `:shields-omgbbq:`** are used to indicate someone's level of happiness (or lack thereof) with their current role. Instead of being used as reacji, people usually add them into their profile status so that it's visible next to their name or username. 
```

Comment:
Should there be a reference to meaning of up vs down or just a link to https://randsinrepose.com/archives/shields-down/?

Reply:
Good point - I can add a link to the blog post!

Reply:
Added!